.devcontainer.json

```diff
@@ -6,7 +6,7 @@
     "vsls-contrib.codetour",
     "esbenp.prettier-vscode"
   ],
-  "postCreateCommand": "npm install --prefix=banking-app && npm install --prefix=auditable-logging-app",
+  "postCreateCommand": "npm install --prefix=banking-app && npm install --prefix=auditable-logging-app && cp sandbox.sh /opt/ccf/bin/sandbox.sh",
```

Comment:
Is there any issue to remove local sandbox.sh? If not, could you raise one?

Reply:
The local sandbox.sh in this repository is the same as here: -https://github.com/microsoft/CCF/commit/77fc334d87fbf3f8e13ace54a4bda391f85e3f20.

When this sandbox enters the dev container we will delete this bit of the postCreateCommand.

How do we get the latest version? We are already on mcr.microsoft.com/ccf/app/dev:3.0.0-dev6-sgx which is the latest dev container?

Reply:
Sorry after looking at my comment again, I found it not clear.
I meant that I want to have a github issue not to forget to delete it.

We can only use the Sandbox of https://github.com/microsoft/CCF/pull/4628 only after it's included in a release.

Reply:
Captured in #60